controllers/deleteComments.controller.js

```diff
@@ -0,0 +1,14 @@
+const removeComments = require('../models/deleteComments.model')
+
+function deleteComments(req, res, next) {
+    const { comment_id } = req.params
+
+    removeComments(comment_id).then(() => {
+        res.status(204).send({message: 'No Content'})
```

Comment:
We should in fact be sending specifically nothing on a delete request. Just a status code